.rubocop.yml

```diff
@@ -13,3 +13,11 @@ inherit_mode:
 Style/NumericLiterals:
   Exclude:
     - db/schema.rb
+
+Style/BlockDelimiters:
+  Exclude:
+    - app/models/feature_flag.rb
+
+Style/MethodCalledOnDoEndBlock:
+  Exclude:
+    - app/models/feature_flag.rb
```

Comment:
Should this be across the project to prevent Rubocop and Prettier arguing about it in the future?

Also I agree we should defer to Prettier because it does more.